buckets/archive/tracker.go

```diff
@@ -88,7 +89,20 @@ func (t *Tracker) run() {
 
 						ctx, cancel := context.WithTimeout(t.ctx, time.Second*5)
 						defer cancel()
-						reschedule, cause, err := t.trackArchiveProgress(ctx, a.BucketKey, a.DbID, a.DbToken, a.JID, a.BucketRoot)
+
+						var ffsInfo *mdb.FFSInfo
+						if account, err := t.colls.Accounts.Get(ctx, a.Owner); err == nil {
+							ffsInfo = account.FFSInfo
+						} else if user, err := t.colls.Users.Get(ctx, a.Owner); err == nil {
+							ffsInfo = user.FFSInfo
+						}
+
+						if ffsInfo == nil {
+							log.Errorf("no FFSInfo, could not find an Account or User: %s", err)
+							return
+						}
```

Comment:
I think here we should call `.Finalize` since looks like this will fail forever.
e.g:
```
if err := t.colls.ArchiveTracking.Finalize(ctx, a.JID, "huh? no FFS info, weird"); err != nil {
        log.Errorf("finalizing errored/rescheduled archive tracking: %s", err)
}
```